docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -806,17 +924,44 @@ Joins two strings.
 
 [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/json.js).
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=json).
+### {{occurrences}}
 
-### {{replace}}
+```handlebars
+{{occurrences str substring}}
+```
+Returns the number of occurrences of substring within the given string. 
+
+#### Parameters

```

Comment:
@junedkazi Can you please provide parameters for this helper?

Reply:
I don't think `occurrences` has any parameters except for `str` and `substring` which we pass to find the occurence.